ZIGSIMPlus/Views/CommandSelectionViewController.swift

```diff
@@ -273,14 +272,35 @@ extension CommandSelectionViewController: UITableViewDataSource {
     }
     
     func setAvailable(_ isAvailable: Bool, forCell cell: StandardCell?) {
-        cell?.commandOnOff.isEnabled = isAvailable
-        cell?.detailButton.isEnabled = isAvailable
+        cell?.commandOnOffButton.isEnabled = isAvailable
         if isAvailable {
             cell?.commandLabel.textColor = Theme.main
-            cell?.detailButton.strokeColor = Theme.main
+            cell?.detailButton.tintColor = Theme.main
+            setDetailImageView(true, forCell: cell)
         } else {
             cell?.commandLabel.textColor = Theme.dark
-            cell?.detailButton.strokeColor = Theme.dark
+            cell?.detailButton.tintColor = Theme.dark
+            setDetailImageView(false, forCell: cell)
+        }
+    }
+    
+    func setDetailImageView(_ isAvailable: Bool,forCell cell: StandardCell? ) {
```

Comment:
Does this method do anything else than setting ImageView?
Then you should consider renaming.
Also `isAvailable` shouldn't be `_` in this case, because it's not understandable from this method name.

Reply:
- ImageView以外も関わっているので、setDetailButtonという名前に変更しました。
- isAvailableという引数名は、実際はコマンドが利用できるかどうかを見ているので、isCommandAvailableに変更しました。また`_` は外しました。
- if分の条件式が長くなっていたので、メソッド化しました。

https://github.com/1-10/ZIGSIMPlus_iOS_Public/pull/60/commits/cb14ab0d50b3dfe66dab747c2570a9ea0d9636ce
https://github.com/1-10/ZIGSIMPlus_iOS_Public/pull/60/commits/b9f215d8be8a677b5d2dea8144a8a249c5b6842d